src/azul/indexer/index_controller.py

```diff
@@ -169,7 +169,7 @@ def aggregate(self, event):
                 log.info('Deferring aggregation of %i contribution(s) to entity %s/%s',
                          tally.num_contributions, tally.entity.entity_type, tally.entity.entity_id)
             entries = [dict(tally.to_message(), Id=str(i)) for i, tally in enumerate(deferrals)]
-            self._document_queue.send_messages(Entries=entries)
+            self._tallies_queue(retry=retry).send_messages(Entries=entries)
```

Comment:
I am unable to determine with ease if retry is ever `True`, may I suggest that you separate this commit into 2 commits minimum? 
One commit could be the refactor and the other the optimization you introduce.

Reply:
It needs to be in one commit, there's no way to make my change without doing a little bit of refactoring.

Reply:
I'm wrong, I forgot I refactored all of the queue names :facepalm:. I'll do that in a separate commit.